DependencyInjection/Configuration.php

```diff
@@ -479,6 +479,21 @@ private function addExceptionSection(ArrayNodeDefinition $rootNode)
                                 })
                             ->end()
                         ->end()
+                        ->booleanNode('forward')
```

Comment:
please add an info node with a description what this does

Reply:
👍 done